### PR TITLE
more relay hosting tweaks

### PR DESCRIPTION
**Pull request by bnewbold**
This has some low-impact doc and scripts changes, and a couple actually-impactful changes to Relay code:

- I think fixes a bug/type with per-PDS, per-second event rate limits
- wires through some concurrency and limit values as env vars; should not impact the default behaviors
- updates PDS resync numbers (in database and results from API) much more frequently, but doesn't log for all those updates. only impacts actual PDS resync processes

### CODE REVIEW

File: bgs/bgs.go [hunk 1]
```diff
@@ -106,7 +106,25 @@ type SocketConsumer struct {
...
+	return BGSConfig{
+		SSL:               true,
+		CompactInterval:   4 * time.Hour,
+		DefaultRepoLimit:  100,
+		ConcurrencyPerPDS: 100,
+		MaxQueuePerPDS:    1_000,
+	}
+}
+
+func NewBGS(db *gorm.DB, ix *indexer.Indexer, repoman *repomgr.RepoManager, evtman *events.EventManager, didr did.Resolver, rf *indexer.RepoFetcher, hr api.HandleResolver, config BGSConfig) (*BGS, error) {
```
Comment (ericvolp12) [inline bgs/bgs.go:127]: This should probably receive the config as a pointer and then init it as the default if the config it receives is nil.

File: bgs/bgs.go [hunk 2]
```diff
@@ -1158,7 +1179,7 @@ func (s *BGS) createExternalUser(ctx context.Context, did string) (*models.Actor
 		peering.Host = durl.Host
 		peering.SSL = (durl.Scheme == "https")
 		peering.CrawlRateLimit = float64(s.slurper.DefaultCrawlLimit)
-		peering.RepoLimit = s.slurper.DefaultPerSecondLimit
+		peering.RateLimit = float64(s.slurper.DefaultPerSecondLimit)
```
Comment (ericvolp12) [inline bgs/bgs.go:1182]: oops, good catch

File: bgs/bgs.go (hunk 1 above)
Comment (bnewbold) [reply bgs/bgs.go]: good point, updated to to that

**Review by ericvolp12 — APPROVED** [review]
sure

**Pull request merged**